lib/transform.js

```diff
@@ -0,0 +1,81 @@
+const vegadataflow = require('../vendor/vega-dataflow.js');
+
+
+var tx = vegadataflow.transforms,
```

Comment:
Actually `var` is still in use in ES6 - I am using it just for as a habit but I will replace with `let` anyway.